tools/nni_annotation/testcase/annotated/dir/simple.py

```diff
@@ -1,13 +1,14 @@
 import nni
 
+
 def max_pool(k):
     pass
 
 
 h_conv1 = 1
-conv_size = nni.choice(2, 3, 5, 7, name='conv_size')
-h_pool1 = nni.function_choice(lambda : max_pool(h_conv1), lambda : avg_pool
-    (h_conv2, h_conv3), name='max_pool')
+conv_size = nni.choice({2: 2, 3: 3, 5: 5, 7: 7}, name='conv_size')
```

Comment:
I am a little confuse of this change. Could you explain?

Reply:
emmmm. Currently it's hard to display the real value in searchspace with nothing changed, so quanlu decided to discard the smartparam temporarily, which has not been public in docs.
I mean, this modification is a temporary move and it is implicit in nni's internal code and will not affect annotation _(which means that the interface of annotation will not be changed)_ but will only affect smartparams _(if you still want to use smartparam, you should write it in this {key: value} format, where the key is the same as the value)_